database/migrations/2021_11_02_230422_create_users_table.php

```diff
@@ -14,13 +14,13 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
-            $table->id();
+            $table->id('users_id');
             $table->string('name');
-            $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
+            $table->string('username');
             $table->string('password');
-            $table->rememberToken();
-            $table->timestamps();
+            $table->string('profile_image');
+            $table->string('role_id');
```

Comment:
I think you need this replaced as `integer` or `unsignedInteger` not as string

Reply:
bukannya di ganti ke foreignId ya?
jadinya begini foreignId('role_id')->constrained('roles');(?)